usaspending_api/references/management/commands/load_disaster_emergency_fund_codes.py

```diff
@@ -84,8 +85,24 @@ def _prep(text):
             return text.strip()
         return text
 
-    def _read_raw_def_code_csv(self):
+    @staticmethod
+    def _prep_date(date_string: str) -> Union[datetime, None]:
+        """Convert a string to a datetime object or None if no string was provided.
+        This assumes the date is in the following format: 12-25-2023 (mm-dd-yyyy)
```

Comment:
Looks like in a few places (including `query_with_filters.py` we use the format `yyyy-mm-dd`. Can you please update this (as well as the CSV) to match?

Reply:
Updated the date format